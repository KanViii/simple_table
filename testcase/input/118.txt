INSERT a number
BEGIN
INSERT b string
INSERT d number
BEGIN
INSERT c number
PRINT
END
END