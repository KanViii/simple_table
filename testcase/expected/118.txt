success
success
success
success
a//0 b//1 d//1 c//2